content/400-reference/100-system-requirements.mdx

```diff
@@ -21,7 +33,38 @@ Notes:
 
 See also: [Supported database versions](database-reference/supported-databases)
 
-</TopBlock>
+### Operating systems
+
+Prisma is supported on MacOS, Windows and most Linux distributions.
+
+#### Linux runtime dependencies
+
+Prisma requires the following system libraries installed to work:
+
+- OpenSSL 1.0.x (except Alpine Linux), 1.1.x or 3.x
+- zlib (`libz.so.1`)
+- libgcc (`libgcc_s.so.1`)
+- C standard library (glibc on most Linux distributions or musl libc on Alpine Linux)
+
+The following table shows the required C standard library version for each supported combination of a Linux distribution, CPU architecture and OpenSSL version:
+
+| Distribution           | CPU architecture | OpenSSL version   | libc version        |
+| ---------------------- | ---------------- | ----------------- | ------------------- |
+| RHEL-like              | x86_64           | 1.0.x, 1.1.x, 3.x | glibc 2.17 or newer |
+| Debian-like            | x86_64           | 1.0.x             | glibc 2.19 or newer |
+| Debian-like            | x86_64           | 1.1.x, 3.x        | glibc 2.24 or newer |
+| RHEL-like, Debian-like | ARM64            | 1.0.x, 1.1.x, 3.x | glibc 2.24 or newer |
+| Alpine Linux           | x86_64, ARM64    | 1.1.x, 3.x        | musl libc 1.2.x     |
```

Comment:
I feel this is far too much information for our standard System Requirements page in our docs. 
This looks scary, and this page should not be scary. 
Is this really needed? Could this be an "extended version" that lives in our repo somewhere and we just link to it here?

Reply:
I actually saw this table in the preview before reading this comment, and my first impression was "wow, I like it".

However we could simplify a bit by splitting the OS support based on the CPU architecture. First we list the major 

And I would specify that we also use `Debian` and `OpenSSL 1.1.x` as fallbacks in case we're not able to detect this info. Should we display the actual warnings Prisma would show in this page as well, so that users googling them would find this page?

Reply:
Yes, that would be the typical "Troubleshooting" entry - first the error message, then explanation, and solution/workaround.